spyder/plugins/ipythonconsole/widgets/status.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Status bar widgets."""
+# Local imports
+from spyder.config.base import _
+from spyder.config.manager import CONF
+from spyder.api.widgets.status import StatusBarWidget
+from spyder_kernels.utils.mpl import MPL_BACKENDS_FROM_SPYDER
+from spyder.api.shellconnect.mixins import ShellConnectMixin
+
+
+class MatplotlibStatus(StatusBarWidget, ShellConnectMixin):
+    """Status bar widget for current matplotlib mode."""
+
+    ID = "matplotlib_status"
```

Comment:
```suggestion
    ID = "matplotlib_status"
    CONF_SECTION = 'ipython_console'
```

With this there's no need to use `CONF` directly here.